docs/setup/yarn_setup.md

```diff
@@ -245,6 +245,18 @@ Note: You can use a different configuration directory per job by setting the env
 
 Note: It is possible to combine `-m yarn-cluster` with a detached YARN submission (`-yd`) to "fire and forget" a Flink job to the YARN cluster. In this case, your application will not get any accumulator results or exceptions from the ExecutionEnvironment.execute() call!
 
+### User jars & Classpath
+
+By default Flink will include the user jars into the system classpath when running a single job. This behavior can be controlled with the `yarn.per-job-cluster.include-job-jar` parameter.
```

Comment:
The configuration parameter is actually called `yarn.per-job-cluster.include-user-jar`.